### ISSUE TITLE
Configurable mB per sip

### ISSUE THREAD

**Issue by ghost** [codewarrior0/sips#3]
I've been playing around with mods that add potion fluids in their own way, namely: Charset Crafting, Inspirations, Immersive Engineering, Thermal Expansion and Gregtech Community Edition. Some of these mods take priority in registering their fluids and most of the time I can store potions in bottles inside tanks, however I cannot bottle them back again easily.  I've found that I had to use Sips to take stored potion fluids in tanks back out so I can drink them, albeit after configuring the effects for each potion fluid. 

Now, you may know that some mods decided that glass bottles only should hold 250 mB, others 333 mB and there's another that sets it at 100 mB. Some of these mods that I've checked have set the bottle to 100 mB without providing a config option. Now I know that each sip reduces the amount in the lil sip/ big chug by 250 mB as described earlier, neither with a configurable amount. Now comes my suggestion:

Can a config option get added where you can configure howmany mB each sip should take? Unless GTCE adds a config option for howmuch glass bottles hold (100 mB), I can right now only turn 5 bottled potions into two sips as GTCE sets glass bottles's capacity to 100 mB each and 5x 100 mB = 500 mB,  enough for two sips. If not, I'll have to compensate for this by making potion effects last 2.5x longer or double the potion strength or have some combination of both. 

I am not sure if you're willing to add this feature to MC 1.12.2 though. Either I'll have to wait for GTCE to add a config option for it or hope for this to get added to this mod instead.  If neither happens, I will slightly buff the effects the potion liquids give me when I use sips.

**Comment by Yoghurt4C** [comment]
I see that forge's unified liquid approach is doing wonders for maintaining consistent volume ratios cross-mod 😭
on a more serious note, I personally consider the quarter-liter sips a good constant that doesn't really need to change (especially due to factors such as setting up a 1.12 devenv is even more of a pain in the ass and that the 1.12 version relies on a lot of hardcoded assumptions, making debugging and extensive testing necessary for my tiny brain); this also isn't really a problem on fabric because most of the community is pushing for arbitrary ratio support to prevent disparities in volumes as much as possible
tl;dr I'll have to say "no", you should pester the current maintainer of gtce since whoever he is, he's probably more likely to listen to trivial demands than arch

**Comment by ghost** [comment]
Ah, I somehow should have expected that setting up a devenv isn't worth it for such a tiny change, especially in an older MC version. I have already mentioned this issue over at GTCE as well with no comment on it as of yet. Oh well. I guess I'll compensate for this by buffing the potion effects slightly until LAGIdiot (current maintainer) adds a config option for it. 

Thanks for the quick response though. This can be closed.
